docs/components/menus.md

```diff
@@ -0,0 +1,353 @@
+<!--docs:
+title: "Menus"
+layout: detail
+section: components
+excerpt: "Menus display a list of choices on temporary surfaces."
+iconId:
+path: /catalog/menus/
+api_doc_root: true
+-->
+
+# Menus
+
+[Menus](https://material.io/components/menus/#) display a list of choices on temporary surfaces.
+
+![Menu hero example showing two cascading menus](assets/menus/menus-hero.png)
+
+# Contents
+
+- [Using menus](#using-menus)
+- [Installing menus](#installing-menus)
+- [Making menus accessible](#making-menus-accessible)
+- [Dropdown menus](#dropdown-menus)
+- [Exposed dropdown menus](#exposed-dropdown-menus)
+- [Theming](#theming)
```

Comment:
I think the table of contents should represent only the top level headers

Reply:
Done